pkg/features/kube_features.go

```diff
@@ -83,6 +83,10 @@ const (
 	//
 	// Enables lending limit.
 	LendingLimit featuregate.Feature = "LendingLimit"
+	// owner: @vicenteferrara
+	// kep: <TODO>
+	// alpha: v0.8
+	DynamicallySizedJobs featuregate.Feature = "DynamicallySizedJobs"
```

Comment:
I like `ResizableJobs` or `ElasticJobs` better

Reply:
Actually ElasticJobs might be confused with ElasticcJobs feature in k/k, so maybe ResizableJobs is better